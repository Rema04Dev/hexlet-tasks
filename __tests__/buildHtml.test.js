import jsBeautify from 'js-beautify';
import buildHtml from '../functional-programming/buildHtml.js';

const { html } = jsBeautify;

test('htmlBuilder', () => {
  expect(html(buildHtml())).toMatchSnapshot();
});
