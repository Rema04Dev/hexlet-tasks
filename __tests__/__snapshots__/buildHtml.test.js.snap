// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`htmlBuilder 1`] = `
"<table>
    <tr>
        <td>lang</td>
        <td>comment</td>
    </tr>
    <tr>
        <td>php</td>
        <td>statements</td>
    </tr>
    <tr>
        <td>clojure</td>
        <td>expressions</td>
    </tr>
</table>"
`;